jetty-http/src/main/java/org/eclipse/jetty/http/HttpCookie.java

```diff
@@ -448,6 +460,39 @@ public static SameSite getSameSiteFromComment(String comment)
         return null;
     }
 
+    /**
+     * Get the default value for SameSite cookie attribute, if one
+     * has been set for the given context.
+     * 
+     * @param context the context to check for default SameSite value
+     * @return the default SameSite value or null if one does not exist
+     * @throws IllegalStateException if the default value is not a permitted value
+     */
+    public static SameSite getSameSiteDefault(ServletContext context)
+    {
+        if (context == null)
+            return null;
+        Object o = context.getAttribute(SAME_SITE_DEFAULT_ATTRIBUTE);
+        if (o == null)
+        {
+            LOG.warn("No default value for SameSite");
+            return null;
+        }
+        
+        if (o instanceof SameSite)
+            return (SameSite)o;
+        
+        try
+        {
+            return Enum.valueOf(SameSite.class, ((String)o).trim().toUpperCase(Locale.ENGLISH));
```

Comment:
Should we set the converted value back so that every cookie doesn't need to carry this cost???